addons/sale_pdf_quote_builder/controllers/quotation_document.py

```diff
@@ -33,7 +37,8 @@ def upload_document(self, ufile, sale_order_template_id=False):
                     'name': ufile.filename,
                     'mimetype': mimetype,
                     'raw': ufile.read(),
-                    'quotation_template_ids': sale_order_template_id,
+                    'quotation_template_ids': sale_order_template.ids,
```

Comment:
```suggestion
                    'quotation_template_ids': sale_order_template_id,
```
keep the line as it was, to reduce diff and keep the blame since there is no reason to change that.

Reply:
Reasons to change it:
1. `*_id` variables should be used for actual ids, not recordsets.
2. allows us to keep this as a single line below 100 chars in length 🙃:
    ```python
            company = sale_order_template.company_id if sale_order_template else request.env.company
    ```

Reply:
Oooh, my bad, I thought it held an id and didn't change :man_facepalming:
Should have read it more precisely above ...